TheMET/TheMET/ViewControllers/CategoryViewController.swift

```diff
@@ -184,14 +186,20 @@ class CategoryViewController: UIViewController, UITableViewDelegate, UITableView
             cell.backgroundColor = .clear
             cell.selectionStyle = .none
             let art = arts[indexPath.row]
-            self.loadCellImage(cell: cell, art: art)
-            cell.isLiked = self.isArtLiked(art: art)
-            cell.text = String.artDescriptionText(art: art)
-            if let departmentName = art.department {
-                cell.tags = [departmentName]
-            }
-            cell.onLikeButtonDidTap = { [weak self] in
-                self?.likeButtonDidTap(cell: cell, art: art)
+            switch art.artData {
+            case .placeholder:
+                cell.isPlaceholderVisible = true
```

Comment:
cell is missing image-loading configuration.
`fun loadCellImage()` is using `tag` to mark each cells to prevent the wrong picture to be displayed. Placeholded cells currently brake this logic.

Reply:
+